README.md

```diff
@@ -36,7 +36,9 @@ Although most of the plugin code was written when an earlier spec was current:
 It also implements the FileWriter spec :
 [http://dev.w3.org/2009/dap/file-system/file-writer.html](http://dev.w3.org/2009/dap/file-system/file-writer.html)
 
-For usage, please refer to HTML5 Rocks' excellent [FileSystem article.](http://www.html5rocks.com/en/tutorials/file/filesystem/)
+>*Note* For Cordova clients that run in native WebViews (for example, iOS and Android) or as native apps (Windows), many of the W3C APIs FileSystem are supported. This is true for the platforms listed in the _Supported Platforms_ list except the Browser platform. On the Browser platform, many of these APIs are deprecated.
```

Comment:
"W3C APIs FileSystem" => "W3C FileSystem APIs"

Avoid the term "native apps" to refer to Cordova apps on Windows. Even though Windows has built-in support for HTML/JS apps, most developers would not consider Cordova apps to be "native". (also on line 562)

How about something like this:
 While the W3C FileSystem spec is deprecated for web browsers, the FileSystem APIs are supported in Cordova applications with this plugin for the platforms listed in the Supported Platforms list, with the exception of the Browser platform.